eng/pipelines/code-quality-reports.yml

```diff
@@ -64,6 +66,13 @@ jobs:
           SkipDefaultCheckout: true
           Paths: $(CheckoutDirectories)
 
+      - task: PowerShell@2
+        inputs:
+          filePath: $(System.DefaultWorkingDirectory)/eng/pipelines/scripts/Get-Linting-Commands.ps1
+          arguments: -BuildReason $(Build.Reason) -SourceBranch "HEAD" -TargetBranch "origin/$("$(System.PullRequest.TargetBranch)" -replace "refs/heads/")" -LintingPipelineVariable "LintingGoals"
```

Comment:
I recommend line folding here so it is easier to read this. (i.e. `>`) to split across lines